client/components/mma/holiday/HolidayReview.tsx

```diff
@@ -212,17 +217,12 @@ export const HolidayReview = () => {
 					</div>
 				) : (
 					<div
-						css={[
-							buttonBarCss,
-							{
-								justifyContent: 'flex-end',
-								marginTop: '20px',
-								[until.mobileMedium]: {
-									flexDirection: 'column',
-									marginTop: 0,
-								},
+						css={{
+							marginTop: '20px',
+							[until.mobileMedium]: {
+								marginTop: 0,
```

Comment:
just fixing a very small layout issue where the inline error component was displaying inline instead of block